src/pages/workspace/WorkspaceInviteMessagePage.js

```diff
@@ -254,9 +260,9 @@ export default compose(
         invitedEmailsToAccountIDsDraft: {
             key: ({route}) => `${ONYXKEYS.COLLECTION.WORKSPACE_INVITE_MEMBERS_DRAFT}${route.params.policyID.toString()}`,
         },
-        savedWelcomeMessage: {
-            key: `${ONYXKEYS.FORMS.WORKSPACE_INVITE_MESSAGE_FORM}Draft`,
-            selector: (draft) => (draft ? draft.welcomeMessage : ''),
+        workspaceInviteMessageDraft: {
+            key: ({route}) => `${ONYXKEYS.COLLECTION.WORKSPACE_INVITE_MESSAGE_DRAFT}${route.params.policyID.toString()}`,
+            selector: (draft) => (isEmpty(draft) ? '' : draft),
```

Comment:
Coming from https://github.com/Expensify/App/issues/32835#issuecomment-1851357992

A selector isn't required here and caused a bug ^